ACCS-365: Payment Services: Stored Cards at Checkout (#821)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins-b2b/requisition-list/containers/requisition-list-selector.mdx

| `sku` | `string` | Yes | Specifies the product SKU to add to the selected requisition list. Required to identify the exact product variant being added. Must match a valid product SKU in your catalog. |
| `selectedOptions` | `string[]` | No | Provides an array of selected product option IDs for configurable products. Captures variant selections such as size, color, or other configurable attributes. Required for configurable products to identify the specific variant being added. |
| `quantity` | `number` | No | Sets the quantity of the product to add to the requisition list. Defaults to 1 if not specified. Use to allow bulk additions or pre-populate quantities from previous orders or saved preferences. |

> | `matchBySKU` | `boolean` | No | Controls how the active state is determined: If set to `true`, it only checks the product SKU. If set to `false`, it checks both the SKU and the selected configurable option UIDs. By default, it uses SKU-only matching (true). Use `false` on product detail pages (PDP) for configurable products so the button is only active when the exact variant (same SKU and selected options) is in the requisition list. Use `true` on product listing pages (PLP), where specific variants can’t be selected. |

Reviewer: ```suggestion
| `matchBySKU` | `boolean` | No | Controls how the active state is determined. When set to `true`, it checks only the product SKU. When set to `false`, it checks both the SKU and selected configurable option UIDs. The default is `true` (SKU-only). Use `false` on product detail pages (PDP) for configurable products so the button is active only for the exact variant (matching SKU and selected options). Use `true` on product listing pages (PLP), where specific variants aren’t selected. |
```

> | `beforeAddProdToReqList` | `function` | No | Callback function to handle validation before the Add to Requisition List dropdown opens when the button is clicked. Use to validate if the product can be added directly (for example, check if a configurable product needs options selected) and redirect to the product detail page if needed. If the callback throws an error or rejects, the dropdown will not open, enabling patterns like redirecting complex products to their detail pages. |

Reviewer: ```suggestion
| `beforeAddProdToReqList` | `function` | No | Callback function that runs before the **Add to Requisition List** dropdown opens. Use it to validate whether the product can be added directly (for example, ensuring required options are selected). If validation fails—by throwing or rejecting—the dropdown will not open, allowing you to handle cases like redirecting configurable products to the product detail page. |
```
